stepflow-rs/tests/mcp/stepflow-config.yml

```diff
@@ -0,0 +1,9 @@
+plugins:
+  - name: builtin
+    type: builtin
+  - name: test-mcp
+    type: mcp
+    command: echo
```

Comment:
Does this actually work? It feels like this is an echo command, not an echo MCP server. Perhaps use the `mock_mcp_server.py` here?

Reply:
Oh, this just invokes the `echo` command on the mock_mcp_server.py.